ciao-cli/instance.go

```diff
@@ -54,6 +54,7 @@ const (
 	}
 	SSHIP   string                                // Instance SSH IP address
 	SSHPort int                                   // Instance SSH Port
+	OsExtendedVolumesVolumesAttached []string     // list of attached volumes
```

Comment:
I'm surprised the CI tools don't complain about a stutter here.  Is "OsExtendedVolumesAttached" sufficiently descriptive or do we need the 2x Volumes?